content/en/docs/components/pipelines/author-a-pipeline/inputs-outputs-data-passing.md

```diff
@@ -0,0 +1,349 @@
++++
+title = "Task I/O"
```

Comment:
Component I/O ?

Reply:
Good though. That's probably the correct name here. I will update.

In general, it's sometimes a bit ambiguous whether something ought to be called a component or a task in the docs. For example, we often talk about "component run time", but this is more accurate as "task run time", IMO. Usually it's fairly inconsequential.